Deploy MongoDB on Azure Cobalt 100 (#12)
odidev/arm-learning-paths: content/learning-paths/servers-and-cloud-computing/mongodb-on-azure/mongotop.md

---
title: MongoDB Benchmarking with mongotop
weight: 8

### FIXED, DO NOT MODIFY
layout: learningpathall
---

> ## Monitoring MongoDB Performance

Reviewer: ```suggestion
## Monitoring MongoDB Performance using Mongotop
```

    test.system_sessions_bench      5ms     2ms      3ms
    benchmarkDB.testCollection      3ms     1ms      1ms
                 test.atlascli      3ms     1ms      1ms
                admin.atlascli      2ms     1ms      1ms
        benchmarkDB.cursorTest      2ms     1ms      1ms
  config.system_sessions_bench      2ms     1ms      1ms
     config.transactions_bench      2ms     1ms      1ms
    local.system_replset_bench      2ms     1ms      1ms
          admin.system.version      0ms     0ms      0ms

                            ns    total    read    write    2025-08-13T08:03:08Z
                admin.atlascli      2ms     1ms      1ms
        benchmarkDB.cursorTest      2ms     1ms      1ms
    benchmarkDB.testCollection      2ms     1ms      1ms
  config.system_sessions_bench      2ms     1ms      1ms
     config.transactions_bench      2ms     1ms      1ms
    local.system_replset_bench      2ms     1ms      1ms
test.admin_system_version_test      2ms     1ms      1ms
                 test.atlascli      2ms     1ms      1ms
    test.system_sessions_bench      2ms     1ms      1ms
          admin.system.version      0ms     0ms      0ms
```

{{% notice Note %}}

> Benchmarking was performed in both an Azure Linux 3.0 Docker container and an Azure Linux 3.0 virtual machine. The benchmark results were found to be relatively stable.

Reviewer: ```suggestion
Benchmarking was performed in both an Azure Linux 3.0 Docker container and an Azure Linux 3.0 virtual machine, with comparable results.
```
